agb-image-converter/src/lib.rs

```diff
@@ -663,6 +668,16 @@ fn valid_sprite_size(width: u32, height: u32) -> bool {
     }
 }
 
+const OUT_DIR_TOKEN: &str = "$OUT_DIR";
+
+fn get_out_dir(raw_input: &str) -> String {
```

Comment:
given the usage here, should this function just do the replacement?

Reply:
I split it to get then replace in order not to call std::env::var for every file when importing multiple sprites.